src/components/space/charts/ActionsByDayChart.tsx

```diff
@@ -145,7 +145,11 @@ const ActionsByDayChart = (): JSX.Element | null => {
         >
           <CartesianGrid strokeDasharray="2" />
           <XAxis dataKey="date" tick={{ fontSize: 14 }} />
-          <YAxis tick={{ fontSize: 14 }} domain={[0, yAxisMax]} />
+          <YAxis
+            tick={{ fontSize: 14 }}
+            domain={[0, yAxisMax]}
+            orientation={direction === 'rtl' ? 'right' : 'left'}
```

Comment:
Can we use sth similar to that instead? So you can use `direction` directly. https://stackoverflow.com/questions/48496151/render-recharts-chart-in-rtl